jerry-core/ecma/base/ecma-module.c

```diff
@@ -831,17 +933,42 @@ ecma_module_connect_imports (void)
  *         ECMA_VALUE_EMPTY - otherwise
  */
 ecma_value_t
-ecma_module_initialize_current (void)
+ecma_module_initialize (ecma_module_t *module_p) /**< module */
 {
-  ecma_value_t ret_value = ecma_module_connect_imports ();
+  ecma_value_t ret_value = ecma_module_connect_imports (module_p);
 
   if (ecma_is_value_empty (ret_value))
   {
-    ret_value = ecma_module_check_indirect_exports ();
+    ret_value = ecma_module_check_indirect_exports (module_p);
   }
 
   return ret_value;
-} /* ecma_module_initialize_current */
+} /* ecma_module_initialize */
+
+static jerry_value_t ecma_module_parse (ecma_module_t *module_p);
```

Comment:
Hm... should the `ecma_module_parse` return `jerry_value_t`? Wouldn't `ecma_value_t` be better?